packages/core/src/connectivity/scp-cf/xsuaa-service.ts

```diff
@@ -135,6 +136,7 @@ export async function refreshTokenGrant(
 }
 
 /**
+ * @deprecated Since v1.49.0 Use `@sap/xssec` lib instead.
```

Comment:
Instead of deprecating the function right away, I would consider to use the xssec implementation to replace our implementation and see that/whether our tests are passing with this. That way you still have the option to roll back if there are problems down the line.
Also you can identify differences in behavior, because of the assumptions in the tests.

Reply:
Discussed, this function was exported, but should not be used.